### DISCUSSION TITLE
Do features like Windows Credential Provider work on eduMFA?

### DISCUSSION THREAD

**Discussion by gnordli** [eduMFA/eduMFA#115] [Q&A]
Hi.   

Do features like Windows Credential Provider work on eduMFA?

thanks!!
Geoff

**Comment by fritterhoff** [comment]
Sorry for the late response, I missed the question. 

Right now we have not changed that much on the APIs so currently all plugins/tools like the Credential Provide will work. In case of (major) upstream changes we have to check if we can provide some compatibility in the future.
